Google.Api.Generator.Tests/ProtoTests/GrpcServiceConfig/Testing.GrpcServiceConfig/GrpcServiceConfigClient.g.cs

```diff
@@ -34,13 +34,11 @@ public sealed partial class GrpcServiceConfigSettings : gaxgrpc::ServiceSettings
         /// <item><description>Initial retry delay: 500 milliseconds.</description></item>
         /// <item><description>Retry delay multiplier: 1.3</description></item>
         /// <item><description>Retry maximum delay: 5000 milliseconds.</description></item>
-        /// <item><description>Initial timeout: 20000 milliseconds.</description></item>
-        /// <item><description>Timeout multiplier: 1</description></item>
-        /// <item><description>Timeout maximum delay: 20000 milliseconds.</description></item>
-        /// <item><description>Total timeout: 20 seconds.</description></item>
+        /// <item><description>Maximum attempts: 2147483647</description></item>
```

Comment:
This should probably say "Maximum attempts: unlimited" or similar.

Reply:
Done.